### DISCUSSION TITLE
Why are my analog ROIs off?

### DISCUSSION THREAD

**Discussion by magicus** [jomjol/AI-on-the-edge-device#3222] [Q&A]
I am having a lot of issues with my analog ROIs. Like 80% of the time, they seem to work fine, and then suddenly they are just almost random.

As an example, look at these two from my latest reading:

<img width="350" alt="Screenshot 2024-09-05 at 16 22 44" src="https://github.com/user-attachments/assets/38848e9a-85c6-465d-8e1e-b4aef8373304">

The leftmost needle is pointing to just short of 2, and the rightmost is almost precisely 8. But AotE thinks they are 3.4 and 5.6, respectively.

Since these values jump around from reading to reading, I get negative values and updating stops, etc, so it causes a lot of trouble.

What could be wrong?

Have I selected the wrong type? Is the alignment bad? Lighting? Focus?

**Comment by friedpa** [comment]
Just play around with the Focus and contrast...
If it looks like this:

![grafik](https://github.com/user-attachments/assets/94e49920-a587-49b3-816e-637801ffc656)

you don´t have false readings at all.
